scss/settings/_component_variables.scss

```
@@ -181,6 +181,12 @@ $card-title-letter-spacing: -0.01em !default;
 // before Bootstrap's own variables (including $spacer) are defined.
 $moo-card-spacing: 1rem !default;
 $moo-card-spacing-sm: 0.75rem !default;
+// Bootstrap exposes one card-cap background for both headers and footers, but
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)